testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js

```diff
@@ -2387,6 +2389,41 @@ QUnit.module('popup options', {
         assert.roughEqual(getOuterWidth($overlayContent), getOuterWidth($container) / 2, 0.1, 'popup width is correct');
         assert.roughEqual(getOuterHeight($overlayContent), getOuterHeight($container) / 2, 0.1, 'popup height is correct');
     });
+
+    [
+        {
+            component: PopupFull,
+            componentName: 'PopupFull',
+            usePopover: false
+        }, {
+            component: PopoverFull,
+            componentName: 'PopoverFull',
+            usePopover: true
+        }
+    ].forEach(({ component, componentName, usePopover }) => {
+        QUnit.test(`${componentName} defaultOptions should affect dropDownEditor popup(T1133910)`, function(assert) {
+            let defaultHandlerCalled = false;
+
+            component.defaultOptions({
+                options: {
+                    onShowing: () => defaultHandlerCalled = true
+                }
+            });
+
+            $('#lookup').dxLookup({
+                usePopover,
+                opened: true
+            });
+
+            assert.strictEqual(defaultHandlerCalled, true);
+
+            component.defaultOptions({
```

Comment:
I think we should move it to try/catch to be sure defaultOptions is restored. Or create a separate module for the scenario and move restoring to `afterEach`

Reply:
done